test/integration/sign_up_test.rb

```diff
@@ -31,7 +31,7 @@ class SignUpTest < ActionDispatch::IntegrationTest
     }
     follow_redirect!
     assert_includes @response.body, "#{@conference.title} - Call for Participation"
-    assert_includes @response.body, 'Update profile'
+    assert_includes @response.body, 'Update Profile'
```

Comment:
This is probably a translation lookup problem (sign_up is not found and is capitalized).  it worked before refactoring, so you should not change the tests ;-)

Reply:
The same key was used for signup, but tested "Sign Up" and "Sign up"...